lms/static/js/student_account/views/FormView.js

```diff
@@ -190,27 +184,34 @@
                 },
 
                 setErrors: function() {
-                    var $msg = this.$errors.find('.message-copy'),
-                        html = [],
+                    var html = [],
                         errors = this.errors,
                         i,
-                        len = errors.length;
+                        len = errors.length,
+                        $errorsContainer;
+
+                // Clear out the old errors if present
+                    this.$formFeedback.find('.submission-error').remove();
 
                     for (i = 0; i < len; i++) {
                         html.push(errors[i]);
                     }
+                    HtmlUtils.append(this.$formFeedback, HtmlUtils.template(this.errorsTpl)({
+                        context: {
+                            title: this.errorsTitle,
+                            messagesHtml: HtmlUtils.HTML(html.join(""))
+                        }
+                    }));
 
-                    $msg.html(html.join(''));
-
-                    this.element.show(this.$errors);
+                    $errorsContainer = this.$formFeedback.find('.submission-error');
 
                 // Scroll to error messages
                     $('html,body').animate({
-                        scrollTop: this.$errors.offset().top
+                        scrollTop: $errorsContainer.offset().top
                     }, 'slow');
 
-                // Focus on first error field
-                    this.focusFirstError();
+                // Focus on the errors container to ensure screen readers see them.
+                    $errorsContainer.focus();
```

Comment:
After testing these pages in various screen reader/browser combos and talking with @cptvitamin , it was decided that the experience would be better for screen reader users if we moved focus to the errors container after form submission.